Resources/Prototypes/_Floof/Entities/Objects/Fun/toys.yml

```
@@ -631,6 +631,285 @@
+    sound:
+      path: /Audio/Items/Toys/mousesqueek.ogg
+      params:
+        volume: 2
+  - type: MeleeWeapon
+    soundHit:
+      path: /Audio/Animals/mouse_squeak.ogg
+      params:
+        variation: 0.05
+        volume: 1.2
+
+- type: entity
+  parent: PlushieHuman
+  id: PlushieERTFox
+  name: ERT fox plushie
+  description: Smells faintly of coffee and sterile surgical tools. She looks tired..
+  components:
+  - type: Sprite
+    sprite: _Floof/Objects/Fun/floof_plush.rsi
+    state: ertfox
+  - type: UseDelay
+    delay: 3.0
+  - type: EmitSoundOnUse
+    sound:
+      path: /Audio/Items/hypospray.ogg
+      params:
+        volume: 2
+  - type: EmitSoundOnLand
+    sound:
+      path: /Audio/_Floof/Voice/Vulpkanin/yap.ogg
+  - type: EmitSoundOnActivate
+    sound:
+      path: /Audio/Items/hypospray.ogg
+      params:
+        volume: 2
+  - type: MeleeWeapon
+    soundHit:
+      path: /Audio/_Floof/Voice/Vulpkanin/yap.ogg
+      params:
+        variation: 0.05
+        volume: 4
+
+- type: entity
+  parent: PlushieHuman
+  id: PlushieRescueFox
+  name: Rescue fox plushie
+  description: These plushies always struggle to stand upright due to misbalancing but their tails are very soft! Wait, doesn't this look familiar to someone...?
+  components:
+  - type: Sprite
+    sprite: _Floof/Objects/Fun/floof_plush.rsi
+    state: parafox
+  - type: UseDelay
+    delay: 3.0
+  - type: EmitSoundOnUse
+    sound:
+      path: /Audio/_Floof/Voice/Vulpkanin/yap.ogg
+      params:
+        volume: 2
+  - type: EmitSoundOnLand
+    sound:
+      path: /Audio/Voice/Vulpkanin/vulpscream3.ogg
+  - type: EmitSoundOnActivate
+    sound:
+      path: /Audio/_Floof/Voice/Vulpkanin/yap.ogg
+      params:
+        volume: 2
+  - type: MeleeWeapon
+    soundHit:
+      path: /Audio/Voice/Vulpkanin/vulpscream3.ogg
+      params:
+        variation: 0.05
+        volume: 4
+
+- type: entity
+  parent: PlushieHuman
+  id: PlushieArachneNobility
+  name: Arachne Nobility Plushie
+  description: Soft as spider silk arachne plushie here to to be of assistance. Do not poke, submit to them poking you instead!
+  components:
+  - type: Sprite
+    sprite: _Floof/Objects/Fun/floof_plush.rsi
+    state: noblearachne
+  - type: UseDelay
+    delay: 3.0
+  - type: EmitSoundOnUse
+    sound:
+      path: /Audio/Voice/Human/womanlaugh.ogg
+      params:
+        volume: 2
+  - type: EmitSoundOnLand
+    sound:
+      path: /Audio/Voice/Arachnid/arachnid_laugh.ogg
+  - type: EmitSoundOnActivate
+    sound:
+      path: /Audio/Voice/Human/womanlaugh.ogg
+      params:
+        volume: 2
+  - type: MeleeWeapon
+    soundHit:
+      path: /Audio/Voice/Arachnid/arachnid_laugh.ogg
+      params:
+        variation: 0.05
+        volume: 4
+
+- type: entity
+  parent: PlushieHuman
+  id: PlushieEldritchCaptain
+  name: Eldritch captain plushie
+  description: This odd plushie semi seems to be watching you somehow, and its eyes glow in the dark? Wait why can it do th-
```

Assuming you want to keep the idea of uncertainty/confusion in the description:
```suggestion
  description: This odd plushie semi-seems to be watching you somehow, and its eyes glow in the dark... Wait, why can it do th-
```